content/en/getting_started/containers/datadog_operator.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Getting Started with the Datadog Operator
+kind: documentation
+further_reading:
+- link: "/agent/kubernetes/integrations/"
+  tag: "Documentation"
+  text: "Create and load an Autodiscovery Integration Template"
+- link: "/agent/guide/ad_identifiers/"
+  tag: "Documentation"
+  text: "Match a container with the corresponding Integration Template"
+- link: "/agent/guide/autodiscovery-management/"
+  tag: "Documentation"
+  text: "Manage which Container to include in the Agent Autodiscovery"
+- link: "/agent/kubernetes/tag/"
+  tag: "Documentation"
+  text: "Dynamically assign and collect tags from your application"
+- link: "/integrations/faq/integration-setup-ecs-fargate/?tab=rediswebui"
+  tag: "faq"
+  text: "Integration Setup for ECS Fargate"
+- link: "/agent/guide/secrets-management/"
+  tag: "Documentation"
+  text: "Secrets Management"
+---
+
+This guide describes the Datadog Operator, how to install it, and how to use it to install the Datadog Agent on Kubernetes.
+
+## What is the Datadog Operator? 
+
+The Datadog Operator is an open source [Kubernetes Operator][4] that enables you to deploy and configure the Datadog Agent in a Kubernetes environment. By using the Operator, you can use a single Custom Resource Definition (CRD) to deploy the node-based Agent, Cluster Agent, and Cluster Checks Runners. The Operator reports deployment status, health, and errors in the Operator's CRD status. Because the Operator uses higher-level configuration options, it limits the risk of misconfiguration.
+
+Once you have deployed the Agent, the Datadog Operator provides:
+
+- Validation for your Agent configurations
+- Keeping all Agents up to date with your configuration
+- Orchestration for creating and updating Agent resources
+- Reporting of Agent configuration status in the Operator's CRD status
+- Optionally, use of an advanced DaemonSet deployment by using Datadog's [ExtendedDaemonSet][5].
+
+<div class="alert alert-warning">The Datadog Operator is in beta.</div>
+
+## Why use the Datadog Operator instead of a Helm chart or DaemonSet?
+
+You can also use a Helm chart or a DaemonSet to install the Datadog Agent on Kubernetes. However, using the Datadog Operator offers the following advantages:
+
+- The Operator has built-in defaults based on Datadog best practices.
+- Operator configuration is more flexible for future enhancements.
+- As a [Kubernetes Operator][4], the Datadog Operator is treated as a first-class resource by the Kubernetes API.
+- Unlike the Helm chart, the Operator is included in the Kubernetes reconciliation loop.
+
+Datadog fully supports using a DaemonSet to deploy the Agent, but manual DaemonSet configuration leaves significant room for error. Therefore, using a DaemonSet is not highly recommended.
+
+## Prerequisites
+
+- Kubernetes Cluster v1.14.X+
```

Comment:
This line should be either
- Kubernetes v1.14.X+
OR
- A Kubernetes cluster running Kubernetes v1.14.X+
if you want to be really pedantic

It's the Kubernetes software that is versioned, not the clusters themselves.